lib/timetable-form/Form.tsx

```diff
@@ -40,18 +61,28 @@ export const Form = (props: React.HTMLAttributes<HTMLDivElement>) => {
       <div className={styles.formRow}>
         <TextInput
           label="Local planning authority"
-          onChange={setLpa}
-          value={lpa}
+          onChange={(lpa) =>
+            setDevelopmentPlan((prev) => ({
```

Comment:
Looking at the way this setter is being used, I wonder if it makes sense to not keep the headers and events in the same state value?